Sources/NostrSDK/Relay.swift

```diff
@@ -51,13 +51,17 @@ public protocol RelayOperating {
     var events: PassthroughSubject<RelayEvent, Never> { get }
 }
 
-/// An optional interface for receiving state updates and events
+/// An optional interface for receiving state updates and responses from relays.
 public protocol RelayDelegate: AnyObject {
     func relayStateDidChange(_ relay: Relay, state: Relay.State)
+
+    func relay(_ relay: Relay, didReceive response: RelayResponse)
+
+    @available(*, deprecated, message: "`relay(Relay, didReceive: RelayEvent)` was replaced by `relay(Relay, didReceive: RelayResponse)` and will be removed shortly.")
```

Comment:
Thoughts on if deprecating this function is the right thing to do?

Reply:
Maybe we could keep it as not deprecated if there is ever a reason to just need the `RelayEvent` and it's a simpler object to interact with, but I don't have a strong opinion about it.